src/ocean/core/BitArray.d

```diff
@@ -86,11 +86,13 @@ struct BitArray
                 uint[] buf = ptr[0 .. olddim];
 
                 buf.length = newdim; // realloc
+                enableStomping(buf);
+
                 ptr = buf.ptr;
-                if( newdim & 31 )
+                if( auto pad_bits = newlen & 31 )
```

Comment:
Well, I think that's the completely oposite of how I read this code :-). So - `pad_bits` is the value of the 31st bit? I think parenthesis makes this more clear IMHO.